zingolib/src/wallet/disk/testing/examples.rs

```diff
@@ -1,161 +1,172 @@
 use super::super::LightWallet;
 
-/// as opposed to [LegacyWalletCase], which enumerates test cases compiled from the history of zingo wallt tests, this ExampleWalletNetworkCase is meant to fully organize the set of test cases.
+/// ExampleWalletNetworkCase sorts first by Network, then seed, then last saved version.
+/// It is public so that any consumer can select and load any example wallet.
 #[non_exhaustive]
 #[derive(Clone)]
-pub enum ExampleWalletNetworkCase {
+pub enum ExampleWalletNetwork {
     /// /
-    Mainnet(ExampleMainnetWalletSeedCase),
+    Regtest(ExampleRegtestWalletSeed),
     /// /
-    Testnet(ExampleTestnetWalletSeedCase),
+    Testnet(ExampleTestnetWalletSeed),
     /// /
-    Regtest(ExampleRegtestWalletSeedCase),
+    Mainnet(ExampleMainnetWalletSeed),
 }
 
 /// /
 #[non_exhaustive]
 #[derive(Clone)]
-pub enum ExampleMainnetWalletSeedCase {
-    /// this is a mainnet seed
-    VTFCORFBCBPCTCFUPMEGMWBP(ExampleVTFCORFBCBPCTCFUPMEGMWBPWalletVersionCase),
+pub enum ExampleMainnetWalletSeed {
+    /// this is a mainnet wallet originally called missing_data_test
+    VTFCORFBCBPCTCFUPMEGMWBP(ExampleVTFCORFBCBPCTCFUPMEGMWBPWalletVersion),
 }
 /// /
 #[non_exhaustive]
 #[derive(Clone)]
-pub enum ExampleVTFCORFBCBPCTCFUPMEGMWBPWalletVersionCase {
+pub enum ExampleVTFCORFBCBPCTCFUPMEGMWBPWalletVersion {
     /// wallet was last saved in this serialization version
     V28,
 }
 /// /
 #[non_exhaustive]
 #[derive(Clone)]
-pub enum ExampleTestnetWalletSeedCase {
-    /// This is a testnet seed.
-    MSKMGDBHOTBPETCJWCSPGOPP(ExampleMSKMGDBHOTBPETCJWCSPGOPPWalletVersionCase),
+pub enum ExampleTestnetWalletSeed {
+    /// This is a testnet seed, generated by fluidvanadium at the beginning of owls (this example wallet enumeration)
+    MSKMGDBHOTBPETCJWCSPGOPP(ExampleMSKMGDBHOTBPETCJWCSPGOPPWalletVersion),
```

Comment:
This is not readable.  Please change the convention so other developers can understand.